src/app/src/main/kotlin/org/imperial/mrc/hint/utils/SecurePaths.kt

```diff
@@ -0,0 +1,21 @@
+package org.imperial.mrc.hint.utils
```

Comment:
Nice to pull these out - maybe it belongs in /security rather than /utils though?

Reply:
You are right, I can have a new package named Security under Utils package.